Index: CMakeLists.txt
===================================================================
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -25,9 +25,7 @@ target_link_libraries(ahpc ${Boost_SYSTEM_LIBRARY} ${OPENSSL_LIBRARIES})
 add_executable(ahps src/http_proxy_server_main.cpp src/http_proxy_server.cpp src/http_proxy_server_config.cpp src/http_proxy_server_connection.cpp src/http_header_parser.cpp src/base64.cpp src/authentication.cpp src/jsonxx/jsonxx.cc)
 target_link_libraries(ahps ${Boost_SYSTEM_LIBRARY} ${Boost_REGEX_LIBRARY} ${OPENSSL_LIBRARIES})
 
-if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-  target_link_libraries(ahpc atomic)
-endif()
+target_link_libraries(ahpc atomic)
 
 if(UNIX)
   target_link_libraries(ahpc pthread)

